userspace/libsinsp/test/sinsp_with_test_input.h

```diff
@@ -106,6 +106,15 @@ class sinsp_with_test_input : public ::testing::Test {
 		throw std::runtime_error("could not retrieve last event or internal error (event vector size: " + std::to_string(m_events.size()) + std::string(")"));
 	}
 
+	scap_evt* add_event_with_ts(uint64_t ts, uint64_t tid, ppm_event_code event_type, uint32_t n, ...)
```

Comment:
what is the difference between this method and `scap_evt* add_event(uint64_t ts, uint64_t tid, ppm_event_code event_type, uint32_t n, ...)`?

Reply:
None, I think this was a leftover during a refactor.